app/src/main/java/chat/rocket/android/layouthelper/ExtRealmModelListAdapter.java

```diff
@@ -37,6 +41,35 @@ protected void notifyRealmModelItemChanged(int position) {
     notifyItemChanged(position + 1);
   }
 
+  public void setLayoutManager(RecyclerView.LayoutManager layoutManager) {
+    this.layoutManager = layoutManager;
+  }
+
+  protected ListUpdateCallback listUpdateCallback = new ListUpdateCallback() {
+    @Override
+    public void onInserted(int position, int count) {
+      notifyItemRangeInserted(position + 1, count);
+      if (layoutManager != null) {
+        layoutManager.scrollToPosition(0);
+      }
```

Comment:
Shall we discuss the feature of autoscroll (or notify new message) in another Pull Request?

This Pull Request is already really worthy just for adding DiffUtils base codes.